yarn-project/circuit-types/src/logs/l1_payload/l1_note_payload.ts

```diff
@@ -31,6 +42,9 @@ export class L1NotePayload extends L1Payload {
     public noteTypeId: Fr,
```

Comment:
Why not have a class similar to the EventSelector?

Reply:
Addressed in later commits